File: Func&String/2941.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "example.txt";
const read = fs.readFileSync(filePath).toString().split("\n");

const source = read[0];
let count = 0;

const addCount = () => (count = count + 1);
for (let i = 0; i < source.length; i++) {
  const e = source[i];
  //č || ć
  if (e === "c") {
    if (source[i + 1] == "=") {
      addCount();
      i = i + 1;
    } else if (source[i + 1] == "-") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  }
  //dž || đ
  else if (e === "d") {
    if (source[i + 1] == "z") {
      if (source[i + 2] == "=") {
        addCount();
        i = i + 2;
      } else {
        addCount();
      }
    } else if (source[i + 1] == "-") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  }
  //lj
  else if (e === "l") {
    if (source[i + 1] == "j") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  }
  //nj
  else if (e === "n") {
    if (source[i + 1] == "j") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  }
  // š
  else if (e === "s") {
    if (source[i + 1] == "=") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  }
  // ž
  else if (e === "z") {
    if (source[i + 1] == "=") {
      addCount();
      i = i + 1;
    } else {
      addCount();
    }
  } else {
    addCount();
  }
}
console.log(count);
//dz=

File: V3/2941.py
import sys
os = sys.platform
if os != "linux":
    print("명재야 보아하니 백준에서 푼게 아니구나")
    input = open("example.txt", "rt").readline
else:
    input = sys.stdin.readline

avg, track_count = map(int, input().split())
""" 만약 (올린 후의)평균이 24가 나왔다면, 올림을 안했을 때의 평균이 23일 때의 저작권 곡에서

딱 1권을 더했을 때의 곡이 23.xxx 가 될 것이고, 이것을 올림하면 24가 될 것이므로,

이 때의 곡의 개수가 평균이 24인 곡의 갯수에서 최소가 될 것이므로, 이것을 구해주면 된다. """
track_count -= 1
melody_count = avg * track_count + 1

print(melody_count)
